data/json/itemgroups/books.json

```diff
@@ -49,10 +49,11 @@
     "items": [
       [ "manual_eskrima", 100 ],
       [ "manual_krav_maga", 100 ],
-      [ "manual_ninjutsu", 100 ],
+      [ "manual_ninjutsu", 50 ],
```

Comment:
Any reason why this was changed from 100 to 50?

Reply:
Out of scope to this PR, sure, but come on. The relative frequency versus the other manuals should be lower to reflect reality.

Reply:
Sneaking in out of scope things in your PRs is a great way to have them stall forever.